src/Testing/ClientFake.php

```diff
@@ -132,6 +134,12 @@ public function record(TestRequest $request): ResponseContract|ResponseStreamCon
         return $response;
     }
 
+    public function responses(): ResponsesContract
+    {
+
+        return new ResponsesTestResource($this);
+    }
```

Comment:
This pattern wise should type the return of the `ResponsesTestResource` instead of the contract. Could we also remove the duplicate newline in it?